src/Scaffolding/VS.Web.CG.Mvc/Templates/Blazor/Index.cs

```diff
@@ -41,8 +41,8 @@ public virtual string TransformText()
 
             this.Write("@page \"/");
             this.Write(this.ToStringHelper.ToStringWithCulture(pluralModelLowerInv));
-            this.Write("\"\r\n\r\n@using Microsoft.EntityFrameworkCore\r\n@using Microsoft.AspNetCore.Components" +
-                    ".QuickGrid\r\n");
+            this.Write("\"\r\n@using Microsoft.EntityFrameworkCore\r\n@using Microsoft.AspNetCore.Components.Q" +
+                    "uickGrid\r\n");
```

Comment:
Is there a reason this string concatenation is here/

Reply:
The template (.tt) associated (.cs) files are autogenerated so I would not worry about the contents of these too much. I always double check the templated content in my tests.